style.css

```diff
@@ -0,0 +1,116 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.myprojects a {
+  text-decoration: none;
+}
+
+#navbar ul {
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+}
+
+#navbar {
+  background-color: #be3144;
+  width: 100%;
+  position: fixed;
+}
+
+#navbar ul li {
+  margin: 0;
+  text-align: right;
+  padding: 1rem;
+}
+
+#navbar a {
+  text-decoration: none;
+  color: #f0f0f0;
+}
+
+#navbar li:hover {
+  background-color: #45567d;
+}
+
+#welcome-section {
+  background-color: #303841;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+#welcome-section h1 {
+  color: #f0f0f0;
+  font-size: 5rem;
+}
+
+#welcome-section p {
+  color: #be3144;
+  font-size: 2rem;
+}
+
+#projects {
+  background-color: #45567d;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  padding: 5rem 1rem;
+}
+
+.project-title {
+  color: #f0f0f0;
+  text-decoration: underline;
+  font-size: 25px;
+  margin-bottom: 25px;
+}
+
+#fleximg {
+  display: grid;
+  grid-auto-columns: auto;
+  grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+  grid-column-gap: 4rem;
+  margin: auto;
+}
+
+#fleximg img {
+  flex: 50%;
+  width: 100%;
+  max-width: 100%;
+  height: 100%;
+  display: block;
+  object-fit: cover;
+}
+
+#fleximg p {
+  background-color: #303841;
+  padding: 1rem;
```

Comment:
You happen to be repeating property values and I'd suggest that you look through your code and group all repeated  in a single class and then use it on the respective HTML elements

Reply:
Well done! :congratulations: